components/Cards.js

```diff
@@ -19,4 +31,93 @@
 //
 // Add a listener for click events so that when a user clicks on a card, the headline of the article is logged to the console.
 //
+
+function createNewCard (articleObject) {
+
+    // create element
+    let card = document.createElement('div');
+    let headline = document.createElement('div');
+    let author = document.createElement('div');
+    let imgContainer = document.createElement('div');
+    let img = document.createElement('img');
+    let byAuthor = document.createElement('span');
+  
+    // assign class
+    card.classList.add('card');
+    headline.classList.add('headline');
+    author.classList.add('author');
+    imgContainer.classList.add('img-container')
+  
+    // assign content
+    headline.textContent = articleObject.headline;
+    img.src = articleObject.authorPhoto;
+    byAuthor.textContent = articleObject.authorName;
+
+    // assign listening event to card, click
+    card.addEventListener('click', () => {
+      console.log(headline)
+    });
+ 
+    //  append to heirarchy 
+    card.appendChild(headline);
+    card.appendChild(author);
+    author.appendChild(imgContainer);
+    author.appendChild(byAuthor);
+    imgContainer.appendChild(img);
+  
+    // append to html
+    let cardsContainer = document.querySelector('.cards-container');
+    cardsContainer.appendChild(card);
+
+};
+
 // Use your function to create a card for each of the articles, and append each card to the DOM.
+
+/*
+object
+
+  data:
+
+    articles:
+
+      bootstrap:  [3 article objects]
+      javascript
+      technology
+      jquery
+      node.js
+
+*/
+let promise = getHerokuArticles();
+
+promise.then(({data: {articles: articles}}) => { // object destructuring   {objectProperty: assignedToVar}
+
+    console.log("Articles object with 5 properties: ", articles);
+
+    // These are 5 arrays
+    let bootstrap = articles.bootstrap;
```

Comment:
You can loop through each of the articles using `Object.entries`
Here's a link explaining more. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

This isn't something you'd be expected to know but I just thought I would mention it since my quick glance of your work looks awesome!